perf-tool/src/monitor.cpp

```diff
@@ -76,6 +77,15 @@ JNIEXPORT void JNICALL MonitorContendedEntered(jvmtiEnv *jvmtiEnv, JNIEnv *env,
     if (numSamples % monitorSampleRate != 0)
         return;
 
+    jvmtiError err;
+    jint hash;
+    err = jvmtiEnv->GetObjectHashCode(object, &hash);
+    if (check_jvmti_error(jvmtiEnv, err, "Unable to retrieve object hashcode.")) {
+	    std::stringstream str;
```

Comment:
Since we typically avoid streams in OpenJ9, could you please replace this code with `sprintf(str, "0x%x", hash)`. 
Also, to me "head" is not descriptive enough. Why not "hash" or "monitorHash"?

Reply:
@mpirvu - fixed.

Reply:
and modified the commit message too, to reflect the new name.